README.md

```diff
@@ -2,7 +2,11 @@
 [![Build Status](https://travis-ci.org/thoughtbot/french_toast.svg?branch=master)](https://travis-ci.org/thoughtbot/french_toast)
 [![Code Climate](https://codeclimate.com/github/thoughtbot/french_toast/badges/gpa.svg)](https://codeclimate.com/github/thoughtbot/french_toast)
 
-Communicate information about background jobs.
+Communicate information about background jobs seamlessly.
+
+FrenchToast gives users the ability to communicate information based on what the user is able to support. If this means that a user is not online, then they should be emailed. If they don't have javascript, they should be able to refresh the page.
+
+Information, should be always be fully rendered on the back end and sent to be displayed on the front end.
```

Comment:
Why is this the case? (I agree with the decision, but the README might be more helpful if it was more explicit)